Plugins/Creature/Creature.cpp

```diff
@@ -3584,3 +3586,56 @@ NWNX_EXPORT ArgumentStack GetMaxAttackRange(ArgumentStack&& args)
 
     return 0.0f;
 }
+
+NWNX_EXPORT ArgumentStack SetMulticlassLimit(ArgumentStack&& args)
+{
+    if (!s_bGetIsClassAvailableHookInitialized)
```

Comment:
No need to check/set a static bool, the static hook below will only be initialized once